yarn-project/end-to-end/src/guides/dapp_testing.test.ts

```diff
@@ -20,86 +24,103 @@ describe('guides/dapp/testing', () => {
       let stop: () => Promise<void>;
       let owner: AccountWallet;
       let recipient: AccountWallet;
-      let token: PrivateTokenContract;
+      let token: TokenContract;
 
       beforeAll(async () => {
         // docs:start:in-proc-sandbox
         ({ rpcServer: rpc, stop } = await createSandbox());
         // docs:end:in-proc-sandbox
         owner = await createAccount(rpc);
         recipient = await createAccount(rpc);
-        token = await PrivateTokenContract.deploy(owner, 100n, owner.getAddress()).send().deployed();
+        token = await TokenContract.deploy(owner).send().deployed();
+        expect((await token.methods._initialize({ address: owner.getAddress() }).send().wait()).status).toEqual(
```

Comment:
style nit - I prefer the old school method where:
```
 const tx = await token.methods._initialize({ address: owner.getAddress() }).send()
const receipt = tx.wait
expect(....)
```

Since this is a guide, might be good to reduce chaining